packages/grafana-data/src/dataframe/DataFrameView.ts

```diff
@@ -23,7 +23,7 @@ export class DataFrameView<T extends object = any> extends FunctionalVector<T> {
   constructor(private data: DataFrame) {
     super();
     const obj = {} as T;
-    const fields = {} as any;
+    const fields: any = {};
```

Comment:
this seems like not much of an improvement. can it at least be like `object` or something like `Partial<Record<string, unknown>>`?